templates/amp/feed.php

```diff
@@ -24,7 +25,11 @@
 
 	<button id="live-list-update-button"
 		update
-		on="tap:amp-live-list-insert-blog.update"
+		<?php if ( $last === false ): ?>
+			on="tap:amp-live-list-insert-blog.update"
+		<?php else: ?>
+			on="tap: AMP.navigateTo(url='<?php echo esc_url( $links->base ); ?>')"
```

Comment:
I don't know React. This has a space after the `tap:`, but the one on line 29 doesn't. Is the inconsistency intentional?

Reply:
I think it makes sense to have no space. When checking the example from an AMP doc it had the space so I assumed it was way to do it. However after doing some searching you don't have to include it and it seems more common to not have the space.